Code/PgSQL/rdkit/CMakeLists.txt

```diff
@@ -172,10 +172,15 @@ string(REGEX REPLACE "'" "" PG_EXTVERSION ${PG_EXTVERSION})
 if(MSVC)
   set(PG_RDKIT_LIB_SRC "${CMAKE_BUILD_TYPE}\\${PG_RDKIT_LIB_SRC}")
 endif(MSVC)
+if(${PostgreSQL_VERSION_STRING} VERSION_LESS 9.5)

```

Comment:
You set it here, but it doesn't look like you actually use `RDKIT_PG_BFP_GIST_FETCH` anywhere.